explorer/templates.go

```diff
@@ -600,5 +600,13 @@ func makeTemplateFuncMap(params *chaincfg.Params) template.FuncMap {
 			return intStr
 		},
 		"floor": math.Floor,
+		"getTotalFromRewardWindowRewardWindowSize": func(idx int, max, t int64) int {
+			x := (max - int64(idx)) * t
+			if x == 0 {
+				return 0
+			}
```

Comment:
The branch seems unnecessary.  It would work as just `return int(time.Duration(x).Seconds())`